README.md

```diff
@@ -49,8 +49,13 @@ Please follow instructions to [configure nydus](./docs/configure_nydus.md) confi
 
 Nydus-snapshotter is implemented as a [proxy plugin](https://github.com/containerd/containerd/blob/04985039cede6aafbb7dfb3206c9c4d04e2f924d/PLUGINS.md#proxy-plugins) (`containerd-nydus-grpc`) for containerd.
 
-A example of starting nydus-snapshotter:
+Assume your server systemd based, install nydus-snapshotter:
+```bash
+make install
+systemctl restart containerd
```

Comment:
Should mention ensuring nydusd/nydus-image have been installed to $PATH?

Reply:
Yes. we can note it here.